source/quick-start.txt

```diff
@@ -64,9 +64,9 @@ Next, create a file to contain your application called ``main.go``
 in the base directory of your project. Use the following sample
 code to run a query on your sample dataset in MongoDB Atlas.
 
-Set the value of the ``uri`` variable with your MongoDB Atlas connection
-string, or create an environment variable with the name ``MONGODB_URI``
-with your Atlas connection string.
+Specify your MongoDB Atlas connection string as the value of the
+``uri`` variable, or create an environment variable called
+``MONGODB_URI`` and set your Atlas connection string as its value.
```

Comment:
Suggestion: You could add a link to the Atlas instructions for creating/finding a connection string.